apps/meteor/app/statistics/server/index.ts

```diff
@@ -7,3 +7,4 @@ export { statistics } from './lib/statistics';
 export { getLastStatistics } from './functions/getLastStatistics';
 export { getStatistics } from './functions/getStatistics';
 export { updateCounter } from './functions/updateStatsCounter';
+export { sendUsageReport } from './functions/sendUsageReport';
```

Comment:
we're avoiding using barrel exports.. we should add it to our docs
```suggestion
```

Reply:
Didn't know that beforehand. Done, thanks!